certbot_dns_godaddy.py

```diff
@@ -17,11 +17,12 @@ class Authenticator(dns_common_lexicon.LexiconDNSAuthenticator):
 
     This Authenticator uses the GoDaddy API to fulfill a dns-01 challenge.
     """
-
     description = ('Obtain certificates using a DNS TXT record (if you are '
                    'using GoDaddy for DNS).')
-    
     def __init__(self, *args: Any, **kwargs: Any) -> None:
+        
+        params_dict = args[0].to_dict()
+        self.ttl = str(params_dict['dns_godaddy_ttl'])
```

Comment:
I think this is better as:
```python
    self.ttl = 600
    if "dns_godaddy_ttl" in params_dict:
        self.ttl = str(params_dict['dns_godaddy_ttl'])
```

It should prevent any error if it's not there and make the option not required.